### PR TITLE
Formatting improvements

### PR DESCRIPTION
**Pull request by nikolaev-rd**
* observability-charter.md contacts & formatting
* README.md formatting

### CODE REVIEW

File: observability-charter.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ and [Umair Ishaq][Umair Ishaq].*
 <!-- TODO: please put github names here -->
 [Alex Nauda]:                 @
 [Alois Reitbauer]:            https://github.com/aloisreitbauer
+[Alolita Sharma]:             @
```
Comment (astencel-sumo) [inline observability-charter.md:34]:
```suggestion
[Alolita Sharma]:             https://github.com/alolita
```

**Comment by halcyondude** [comment]
Thank you for contributing!  Would you please split this into 2 PR's, one for the charter + README.md, and the other for the paper?  

As this updates the contact info for @alolita I would ask that she approve this change.

**Comment by nikolaev-rd** [comment]
> As this updates the contact info for @alolita I would ask that she approve this change.

Her contact already i this document ([line 73](https://github.com/cncf/tag-observability/pull/145/files?diff=unified&w=0#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L73))

**Comment by nikolaev-rd** [comment]
@halcyondude MR splitted, sorry for dummed commits.

File: observability-charter.md [hunk 2]
```diff
@@ -51,7 +52,8 @@ and [Umair Ishaq][Umair Ishaq].*
 ## Introduction
 
 This document describes the purpose and operations of the Cloud Native
-Computing Foundation ([CNCF]) Technical Advisory Group([TAG]) on Observability.
+Computing Foundation ([CNCF]) Technical Advisory Group ([TAG]) on Observability.
+Computing Foundation ([CNCF]) Technical Advisory Group ([TAG]) on Observability.
```
Comment (astencel-sumo) [inline observability-charter.md:56]: Why is this line repeated?

**Review by astencel-sumo — APPROVED** [review]

**Comment by nikolaev-rd** [comment]
So simple changes, and so much time for approve.

**Comment by nikolaev-rd** [comment]
@alolita review please

**Comment by alolita** [comment]
This fix has been made by a different PR. Closing this older PR.

**Pull request closed without merging**